### DISCUSSION TITLE
STAR Major Uncertainty and Recommendation #7: Loss of old female

### DISCUSSION THREAD

**Discussion by brianlangseth-NOAA** [pfmc-assessments/canary_2023#44] [General]
From 2015 report:

Uncertainty: Considerable uncertainty remains regarding the most appropriate method to account for the lack of older females in the stock (either by age-specific change in M for females, or selectivity effects).

Recommendation: Basic life history research may help to resolve assessment uncertainties regarding appropriate values for natural mortality and steepness, and how to best account for the apparent loss of older females in the population.

**Comment by brianlangseth-NOAA** [comment]
This issue will be applicable for black rockfish too. Possibly discuss at pre-assessment workshop along with @shcaba and other black rockfish leads (@aaronmberger-nwfsc  and @EJDick-NOAA). John Field and ODFW biologists were identified as also having opinions on the matter so also worthwhile to include in discussions. 

- [x] @brianlangseth-NOAA committed to organizing the discussion around this issue at the [September 21 PEP meeting](https://docs.google.com/document/d/1-HtrKAHtnvrINMr14cyIKqUZ-ObyL7fD-F2s15JoZc8/edit?pli=1#). Probably best after gathering more understanding of what the issue is. Thus November at the earliest. Seems like there is two main considerations, 1) what is biologically more probable, and 2) based on the answer to the first question, how best to model this issue.

**Comment by EJDick-NOAA** [comment]
For the 2017 [Blue/Deacon Rockfish assessment](https://www.pcouncil.org/documents/2018/01/the-combined-status-of-blue-and-deacon-rockfishes-in-u-s-waters-off-california-and-oregon-in-2017.pdf/), @aaronmberger-nwfsc and I profiled over the apical selectivity parameter while estimating M. See Table 62 and section 2.10.5 in that assessment. This approach could be used to look at sensitivity of the model outputs to alternative hypotheses regarding differences in M and/or selectivity. Note that in that assessment, there were fewer males than females in the observed catch.

**Comment by chantelwetzel-noaa** [comment]
This definitely needs to be investigated within the next assessment since the current parameterization was carried forward in the 2015 assessment within in-depth explorations due to time.  DFO just completed a new assessment (reviewed a few weeks ago) for canary rockfish in British Columbia and I know they were exploring a similar parameterization for female natural mortality. Rowen Haigh (Rowan.Haigh@dfo-mpo.gc.ca) was one of the co-authors and could direct you to any public documents.

**Comment by brianlangseth-NOAA** [comment]
See [notes from two meetings](https://docs.google.com/document/d/1b-_UA3XPMkFnuV8M0bERhGk0KPSiciul8RSpVbWa_RY/edit) in January that disucssed this issue

Conclusions were to use sex specific constant M, but explore as sensitivities.

**Reply by brianlangseth-NOAA** [reply]
ODFW shared report during those meetings, which by the time the assessment comes out will hopefully be published and can be cited.
